### DISCUSSION TITLE
Discord or Reddit channel

### DISCUSSION THREAD

**Discussion by a-gonzalo** [sqlpage/SQLPage#228] [General]
Hi! 

I was wondering why there isn't a Discord or Reddit for SQLPage.

Probably, It would be a great idea for faster or direct answers, but probably we will lose the knowledge that Discussion forum gives.

What's your opinion on this?

**Comment by lovasoa** [comment]
Hello ! We don't have a reddit or discord yet. We will probably get to this when the community grows, but currently, I think we are doing well with github discussions.

Is there a particular issue you are encountering with discussions ?

**Reply by a-gonzalo** [reply]
There isn't an issue as well,  I'm just guessing that having some socials would be positive to the project in order to be known.
Also, a direct chat has (normally) more people talking about the project at the same time.

But the function of the Discussions page (as a perfect repository of documentation) is good too. 
So I was just guessing about it.
